odata-json-format/15 Delta Payload.md

```diff
@@ -478,17 +492,152 @@ inline as a delta
     },
     {
       "@context": "#Customers/$link",
+      "@Org.OData.Core.V1.ContentID": "5",
       "source": "Customers('ANATR')",
       "relationship":" Orders",
       "target": "Orders(10643)"
     },
     {
       "@context": "#Customers/$deletedLink",
+      "@Org.OData.Core.V1.ContentID": "6",
       "source": "Customers('DUMON')",
       "relationship": "Orders",
       "target": "Orders(10311)"
     }
   ]
 }
 ```
+
+Assuming all changes can be applied without errors, the response would be
+```
+HTTP/1.1 204 No Content
+Preference-Applied: return=minimal, continue-on-error
+
+
+
```

Comment:
Why three blank lines? My server produces only one.

Reply:
Actually there have to be two blank lines because the header block of an HTTP message is ended by LF LF. My server and browser shows this when selecting to highlight:
![image](https://github.com/oasis-tcs/odata-specs/assets/951576/48e7aad1-5972-4d76-b814-2d00948dd925)

Which are exactly the two empty lines I want.

Reply:
My web server produces the following:
```
HTTP/1.1 204 No Content <CRLF>
Preference-Applied: return=minimal, continue-on-error<CRLF>
<CRLF>
```
The final two <CRLF>s are at the end of the second and third line. There is no fourth line. Does it look the same for you?

Reply:
The produced HTML looks fine for me:
https://github.com/oasis-tcs/odata-specs/blob/09fb4d4b02c452e58758784047e8421a6643b9d1/docs/odata-json-format/odata-json-format.html#L1397-L1401

Line 1399 is the separator line between the header block and the response body, and line 1400 is the empty response body

Reply:
On the wire it would be `...continue-on-error<LF><LF>`.

Reply:
The code on the wire produces _one_ empty line (the one between the two `<LF>`s). But the generated HTML contains _two_ empty lines.

Reply:
That's a tricky visualization question.

Assume the response is an empty JSON object, on the wire `last: header<LF><LF>{}`. This would be visualized as
~~~http
last: header

{}
~~~

If the response has an empty response body, we'd get on the wire `last: header<LF><LF>` on the wire, just the `{}` removed. So if we remove the curlies also from the visualization, we get
~~~http
last: header


~~~

In both cases the visualization has actually three <LF> octets whereas the message only has two - there is no trailing <LF> required at the end of an HTTP message.